op-node/rollup/derive/batches.go

```diff
@@ -198,7 +199,9 @@ func CheckBatch(cfg *rollup.Config, log log.Logger, l1Blocks []eth.L1BlockRef, l
 					return BatchUndecided
 				}
 				nextOrigin := l1Blocks[1]
-				if batch.Batch.Timestamp >= nextOrigin.Time { // check if the next L1 origin could have been adopted
+				// If Espresso is sequencing, the sequencer cannot adopt the next origin in the case
+				// that HotShot failed to sequence any blocks
```

Comment:
We should maybe modify the rules so that in the case where HotShot has not sequenced any blocks, the sequencer uses the next origin if it is ready else the previous origin. That way we would remain consistent with the existing constraints. This is something that can be done in a followup, though (another `op-integration-loose-ends` issue)

Reply:
Added #47